src/declarations.js

```diff
@@ -1,5 +1,5 @@
 /* @flow */
 
-import type { MuseGlobalType } from './types';
+import type { MuseGlobalType } from './lib/types';
```

Comment:
would it make sense to create a `types` folder and put `MuseGlobalType` in `index.js` for now until we have more types? When I think about `lib` usually I think helper libraries and things like that. I think `types` should be in a folder of its own.

Reply:
Agreed, but I'd prefer to do that + additional refactoring in another PR if possible.